src/composables/accounts.ts

```diff
@@ -26,11 +27,30 @@ export function useAccounts({ store }: IDefaultComposableOptions) {
     return accountsAddressList.value.includes(address);
   }
 
+  /**
+   * This function is used to format the given address.
+   * @param {string} address - the address to format.
+   * @returns {string} - returns the account's name if it exists,
+   * otherwise it returns the translated string "Account" + the account's index +1.
+  */
+  function getAccountNameByAddress(address: string): string {
+    const _account = accounts.value.find(
```

Comment:
I strongly suggest to avoid using underscores as this is less readable. If there is a name collision there are other options. Like for example `accountFound`. Or even better:

```
const { idx, name } = accounts.value.find((acc) => acc.address === address) || {};
```